### PR TITLE
Update storage version

### PR DESCRIPTION
**Pull request by emgerner-msft**

### CODE REVIEW

File: lib/services/storageManagement2/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "Wang, Yugang <yugangw@microsoft.com>",
     "Zavery, Amar <amzavery@microsoft.com>"
   ],
-  "version": "0.12.0",
+  "version": "0.12.0-alpha",
```
Comment (amarzavery) [inline lib/services/storageManagement2/package.json]: I think it should be "0.12.0-preview", as it is a common pattern in the node sdk. I have submitted another PR https://github.com/Azure/azure-sdk-for-node/pull/1595 as I had published new version of the client runtime and wanted to regenerate using a new version of the code generator.

**Comment by amarzavery** [comment]
@emgerner-msft - I think these two branches in upstream named 'emgerner-msft-patch-1' and 'emgerner-msft-patch-2' were created by mistake. You can take a look at them [here](https://github.com/Azure/azure-sdk-for-node/branches). If so, then should I delete them?

**Comment by emgerner-msft** [comment]
I deleted emgernermsft-patch-1 (that was an accident), but patch-2 is this change so probably best not to delete that until merged.

**Comment by emgerner-msft** [comment]
Ah sorry, didn't realize until just now that you updated version in the other pull request. Preview is fine too, the examples I found npm referencing used alpha but it's not important to me. I'll close this. Please mail me when you've merged the other change.
